.circleci/config.yml

```diff
@@ -107,21 +107,23 @@ jobs:
 
   sam_build:
     docker:
-      - image: public.ecr.aws/sam/build-python3.10:latest
+      - image: public.ecr.aws/sam/build-python3.12:latest
     working_directory: ~/repo
     steps:
       - checkout
       - run:
           name: sam build
           command: |
+            dnf install -y postgresql-devel
+            pip install setuptools wheel
```

Comment:
I'm not too worried we now have to install steuptools and wheel here. That kinda tracks.

I'm a bit confused about why we get the `Error: pg_config executable not found` error without installing `postgresql-devel` because we're using psycopg2-binary which should be pre-compiled :confused:

Reply:
This was because the version of psycopg2-binary we were using didn;t have python 3.12 weheels. Now fixed